Blockzilla/SettingsViewController.swift

```diff
@@ -357,23 +365,62 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         let contentViewController = SettingsContentViewController(url: url)
         navigationController?.pushViewController(contentViewController, animated: true)
     }
-
+    
+    @objc func tappedLearnMoreSearchSuggestionsFooter(gestureRecognizer: UIGestureRecognizer) {
+        guard let url = SupportUtils.URLForTopic(topic: "search-suggestions") else { return }
+        let contentViewController = SettingsContentViewController(url: url)
+        navigationController?.pushViewController(contentViewController, animated: true)
+    }
+    
+    fileprivate func getToggleCell(indexPath: IndexPath) -> UITableViewCell {
+        let cell: UITableViewCell = UITableViewCell(style: .subtitle, reuseIdentifier: "toggleCell")
+        let toggle = toggleForIndexPath(indexPath)!
+        cell.textLabel?.text = toggle.label
+        cell.textLabel?.numberOfLines = 0
+        cell.accessoryView = PaddedSwitch(switchView: toggle.toggle)
+        cell.detailTextLabel?.text = toggle.subtitle
+        cell.detailTextLabel?.numberOfLines = 0
+        cell.selectionStyle = .none
+        if toggle.label == UIConstants.strings.labelSendAnonymousUsageData ||
+            toggle.label == UIConstants.strings.settingsSearchSuggestions{
+            let selector = toggle.label == UIConstants.strings.labelSendAnonymousUsageData ? #selector(tappedLearnMoreFooter) : #selector(tappedLearnMoreSearchSuggestionsFooter)
+            let learnMore = NSAttributedString(string: UIConstants.strings.learnMore, attributes: [.foregroundColor : UIConstants.colors.settingsLink])
+            let space = NSAttributedString(string: " ", attributes: [:])
+            guard let subtitle = toggle.subtitle else { return cell }
+            let attributedSubtitle = NSMutableAttributedString(string: subtitle)
+            attributedSubtitle.append(space)
+            attributedSubtitle.append(learnMore)
+            cell.detailTextLabel?.attributedText = attributedSubtitle
+            let tapGesture = UITapGestureRecognizer(target: self, action: selector)
+            cell.addGestureRecognizer(tapGesture)
+        }
+        return cell;
+    }
+    
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         var cell: UITableViewCell
         switch sections[indexPath.section] {
+        case .privacy:
+            if indexPath.row == 0 {
+                cell = SettingsTableViewCell(style: .subtitle, reuseIdentifier: "trackingCell")
+                cell.textLabel?.text = String(format: UIConstants.strings.trackingProtectionLabel)
+                cell.accessibilityIdentifier = "settingsViewController.trackingCell"
+                cell.accessoryType = .disclosureIndicator
+            } else {
+                cell = getToggleCell(indexPath: indexPath)
```

Comment:
Cell should have an accessibilityIdentifier.

Reply:
@oliviabrown9 it doesn't look like our other getToggleCell's in the else conditions are given accessibilityIdentifiers. Perhaps that should be a follow up ticket? It wouldn't make sense to include that in this line anyway, as we'd be giving every other cell the same accessibilityIdentifier.